### PR TITLE
WA-4428: SDK - Create manual transactions

### PR DESCRIPTION
**Pull request by wgkeller**
## The problem ##

As per: 

WA-4428

Points addressed:
* Added new request types
* Created new test case
* Created new API endpoint for POST 
* Cleaned up gradle files to make project build locally without including into parent project

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,4 @@
 .externalNativeBuild
 fastlane/report.xml
 schemas/
+
```
Comment (VindhyaRajagopal) [inline .gitignore:11]: Extra line can be removed?

Comment (wgkeller) [reply .gitignore]: Yes. Done.

File: .gitignore [hunk 2]
```diff
@@ -7,4 +7,4 @@
 /captures
 .externalNativeBuild
 fastlane/report.xml
-schemas/
+schemas/
```
Comment (VindhyaRajagopal) [inline .gitignore:10]: Can we update overview.md & packages.md(for added packages like TransactionType)?

**Review by VindhyaRajagopal — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 2 above)
Comment (wgkeller) [reply .gitignore:10]: Updated overview.md but there are no new packages created, only new types in existing packages

**Review by VindhyaRajagopal — APPROVED** [review]

**Pull request merged**